src/index.js

```diff
@@ -0,0 +1,126 @@
+const { Producer, KafkaClient } = require('kafka-node');
+const logtify = require('logtify');
+
+const { streamBuffer } = logtify;
+const { stream, adapters } = logtify();
+
+/**
+ * Kafka plugin for logtify
+ */
+class Kafka extends stream.Subscriber {
+  constructor(configs) {
+    super();
+    this.ready = false;
+    this.settings = configs || {};
+    this.kafkaClient = undefined;
+    this.kafkaProducer = undefined;
+    if (this.settings.KAFKA_HOST && this.settings.KAFKA_TOPIC) {
+      this.initialize();
+    } else {
+      console.warn('Kafka logtify module is not active due to a missing KAFKA_HOST and/or KAFKA_TOPIC');
```

Comment:
would it be wise to use logtify (with , also, console) to log this? like here https://github.com/dial-once/node-logtify-kafka/pull/1/files#diff-1fdf421c05c1140f6d71444ea2b27638R52

Reply:
Good suggestion. Applied